### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Harshh18**
Added  LOC badge, visitors count badge, stars count badge, forks badge and contributors badge.
Also, made the heading align in center for better viewing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
+<div align="center">
+<h1> Blockchain Developer, Smart Contract, & Solidity Course - Powered By AI - Beginner to Expert Course | Foundry Edition 2023 </h1>
 
-# Blockchain Developer, Smart Contract, & Solidity Course - Powered By AI - Beginner to Expert Course | Foundry Edition 2023
+<a href="https://github.com/Cyfrin/foundry-full-course-f23"><img src="https://sloc.xyz/github/Cyfrin/foundry-full-course-f23" alt="LOC"/></a>
```
Comment (PatrickAlphaC) [inline README.md:4]: Could you remove this?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
+<a href="https://github.com/Cyfrin/foundry-full-course-f23"><img src="https://visitor-badge.laobi.icu/badge?page_id=Cyfrin.foundry-full-course-f23" alt="visitor count"/></a>
```
Comment (PatrickAlphaC) [inline README.md:5]: Could you remove this?

**Review by PatrickAlphaC — CHANGES_REQUESTED** [review]
Thanks for this! Two of them I don't like. Could you remove and I'll merge?

File: README.md (hunk 2 above)
Comment (Harshh18) [reply README.md:5]: Yeah, sure

Comment (Harshh18) [reply README.md:5]: Do you want me to remove the visitors count and Lines of code badges right?

**Review by Harshh18 — COMMENTED** [review]
Removed LOC and visitors count badges.

**Comment by Harshh18** [comment]
Hey, Thank you for suggesting changes, I have removed LOC and Visitors count badges. Is there anything that you'd like to add or change? Ping me up for the same, I'll do it. Thanks and regards,Harsh Khandelwal From: Patrick CollinsSent: 12 July 2023 08:40To: Cyfrin/foundry-full-course-f23Cc: Harsh Khandelwal; AuthorSubject: Re: [Cyfrin/foundry-full-course-f23] Update README.md (PR ***@***.*** requested changes on this pull request.Thanks for this! Two of them I don't like. Could you remove and I'll merge?In README.md:>  -# Blockchain Developer, Smart Contract, & Solidity Course - Powered By AI - Beginner to Expert Course | Foundry Edition 2023+<a href="https://github.com/Cyfrin/foundry-full-course-f23">LOC</a>Could you remove this?In README.md:>  -# Blockchain Developer, Smart Contract, & Solidity Course - Powered By AI - Beginner to Expert Course | Foundry Edition 2023+<a href="https://github.com/Cyfrin/foundry-full-course-f23">LOC</a>+<a href="https://github.com/Cyfrin/foundry-full-course-f23">visitor count</a>Could you remove this?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Pull request merged**